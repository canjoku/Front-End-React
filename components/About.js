import { Component } from 'react'

class About extends Component {
  render () {
    return (
      <main>
        <h1 className="lg-heading">About<span class="text-secondary">Me</span></h1>
        <h2 className="sm-heading">Let me tell you a few things...</h2>

        <div className="about-info">
          <img src="../static/images/pic24.jpg" alt="Chidi Njoku" className="bio-image"/>

          <div className="bio">
            <h3 className="text-secondary">BIO</h3>
            <p>I am a passionate and talented developer. I posess strong coding,
              technical, problem solving abilities and a knack for writting clean, readable code. I am the kind of guy who goes
              beyond the routine and likes to do a little bit of everything. I am committed to collaborative
              problem solving, building quality products and sophisticated design.
            </p>
          </div>

          <div className="job job-1">
            <h3>BackEnd</h3>
            <h6>Full Stack Developer</h6>
            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates perferendis totam enim. Nesciunt porro dolores
              expedita dolor necessitatibus deserunt nemo.
            </p>
          </div>

          <div className="job job-2">
            <h3>FrontEnd</h3>
            <h6>Front End Developer</h6>
            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates perferendis totam enim. Nesciunt porro dolores
              expedita dolor necessitatibus deserunt nemo.
            </p>
          </div>

          <div className="job job-3">
            <h3>Devops</h3>
            <h6>Graphic Designer</h6>
            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates perferendis totam enim. Nesciunt porro dolores
              expedita dolor necessitatibus deserunt nemo.
            </p>
          </div>
        </div>
      </main>
    )
  }
}

export default About