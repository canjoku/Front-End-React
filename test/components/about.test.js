import About from '../../components/About'

test('<About /> renders correctly', () => {
  const component = shallow(<About />)
  expect(component).toMatchSnapshot()
})
