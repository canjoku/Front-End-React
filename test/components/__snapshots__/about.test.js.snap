// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <About />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="u-center-text u-bottom-margin-small"
        >
          <h2
            className="c-about-section__primary-heading"
          >
            About Me
          </h2>
        </div>,
        <div
          className="c-about-section__text-image-container"
        >
          <div
            className="c-image-composition"
          >
            <img
              alt="photo-1"
              className="c-image-composition__photo c-image-composition__photo--p1"
              src="/static/images/pic22.jpg"
            />
            <img
              alt="photo-2"
              className="c-image-composition__photo c-image-composition__photo--p2"
              src="/static/images/pic6.jpg"
            />
            <img
              alt="photo-3"
              className="c-image-composition__photo c-image-composition__photo--p3"
              src="/static/images/pic20.jpg"
            />
          </div>
          <div
            className="c-about-section__about-me"
          >
            <h3
              className="c-about-section__tertiary-heading u-bottom-margin-small"
            >
              I'm Chidi Njoku and I'm a full stack web developer
            </h3>
            <p
              className="u-bottom-margin-medium"
            >
              I am a passionate and talented developer. I posess strong coding, technical, problem solving abilities and a knack for writting clean, readable code. I am the kind of guy who goes beyond the routine and likes to do a little bit of everything. I am committed to collaborative problem solving, building quality products and sophisticated design.
            </p>
            <h3
              className="c-about-section__tertiary-heading"
            >
              Summary of Skills
            </h3>
            <ul
              className="c-about-section__list"
            >
              <li
                className="c-about-section__item"
              >
                Experience with Ruby on Rails
              </li>
              <li
                className="c-about-section__item"
              >
                Solid understanding of SQL
              </li>
              <li
                className="c-about-section__item"
              >
                Experince with AWS and Docker containers
              </li>
              <li
                className="c-about-section__item"
              >
                Experince with Test-Driven Development (TDD), specifically Rspec/Capybara and Jest/Enzyme
              </li>
              <li
                className="c-about-section__item"
              >
                Experience with conitnuous integration
              </li>
              <li
                className="c-about-section__item"
              >
                Experience with GraphQl, React.js
              </li>
            </ul>
          </div>
        </div>,
      ],
      "className": "c-about-section",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h2
            className="c-about-section__primary-heading"
          >
            About Me
          </h2>,
          "className": "u-center-text u-bottom-margin-small",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "About Me",
            "className": "c-about-section__primary-heading",
          },
          "ref": null,
          "rendered": "About Me",
          "type": "h2",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="c-image-composition"
            >
              <img
                alt="photo-1"
                className="c-image-composition__photo c-image-composition__photo--p1"
                src="/static/images/pic22.jpg"
              />
              <img
                alt="photo-2"
                className="c-image-composition__photo c-image-composition__photo--p2"
                src="/static/images/pic6.jpg"
              />
              <img
                alt="photo-3"
                className="c-image-composition__photo c-image-composition__photo--p3"
                src="/static/images/pic20.jpg"
              />
            </div>,
            <div
              className="c-about-section__about-me"
            >
              <h3
                className="c-about-section__tertiary-heading u-bottom-margin-small"
              >
                I'm Chidi Njoku and I'm a full stack web developer
              </h3>
              <p
                className="u-bottom-margin-medium"
              >
                I am a passionate and talented developer. I posess strong coding, technical, problem solving abilities and a knack for writting clean, readable code. I am the kind of guy who goes beyond the routine and likes to do a little bit of everything. I am committed to collaborative problem solving, building quality products and sophisticated design.
              </p>
              <h3
                className="c-about-section__tertiary-heading"
              >
                Summary of Skills
              </h3>
              <ul
                className="c-about-section__list"
              >
                <li
                  className="c-about-section__item"
                >
                  Experience with Ruby on Rails
                </li>
                <li
                  className="c-about-section__item"
                >
                  Solid understanding of SQL
                </li>
                <li
                  className="c-about-section__item"
                >
                  Experince with AWS and Docker containers
                </li>
                <li
                  className="c-about-section__item"
                >
                  Experince with Test-Driven Development (TDD), specifically Rspec/Capybara and Jest/Enzyme
                </li>
                <li
                  className="c-about-section__item"
                >
                  Experience with conitnuous integration
                </li>
                <li
                  className="c-about-section__item"
                >
                  Experience with GraphQl, React.js
                </li>
              </ul>
            </div>,
          ],
          "className": "c-about-section__text-image-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt="photo-1"
                  className="c-image-composition__photo c-image-composition__photo--p1"
                  src="/static/images/pic22.jpg"
                />,
                <img
                  alt="photo-2"
                  className="c-image-composition__photo c-image-composition__photo--p2"
                  src="/static/images/pic6.jpg"
                />,
                <img
                  alt="photo-3"
                  className="c-image-composition__photo c-image-composition__photo--p3"
                  src="/static/images/pic20.jpg"
                />,
              ],
              "className": "c-image-composition",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "photo-1",
                  "className": "c-image-composition__photo c-image-composition__photo--p1",
                  "src": "/static/images/pic22.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "photo-2",
                  "className": "c-image-composition__photo c-image-composition__photo--p2",
                  "src": "/static/images/pic6.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "photo-3",
                  "className": "c-image-composition__photo c-image-composition__photo--p3",
                  "src": "/static/images/pic20.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="c-about-section__tertiary-heading u-bottom-margin-small"
                >
                  I'm Chidi Njoku and I'm a full stack web developer
                </h3>,
                <p
                  className="u-bottom-margin-medium"
                >
                  I am a passionate and talented developer. I posess strong coding, technical, problem solving abilities and a knack for writting clean, readable code. I am the kind of guy who goes beyond the routine and likes to do a little bit of everything. I am committed to collaborative problem solving, building quality products and sophisticated design.
                </p>,
                <h3
                  className="c-about-section__tertiary-heading"
                >
                  Summary of Skills
                </h3>,
                <ul
                  className="c-about-section__list"
                >
                  <li
                    className="c-about-section__item"
                  >
                    Experience with Ruby on Rails
                  </li>
                  <li
                    className="c-about-section__item"
                  >
                    Solid understanding of SQL
                  </li>
                  <li
                    className="c-about-section__item"
                  >
                    Experince with AWS and Docker containers
                  </li>
                  <li
                    className="c-about-section__item"
                  >
                    Experince with Test-Driven Development (TDD), specifically Rspec/Capybara and Jest/Enzyme
                  </li>
                  <li
                    className="c-about-section__item"
                  >
                    Experience with conitnuous integration
                  </li>
                  <li
                    className="c-about-section__item"
                  >
                    Experience with GraphQl, React.js
                  </li>
                </ul>,
              ],
              "className": "c-about-section__about-me",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "I'm Chidi Njoku and I'm a full stack web developer",
                  "className": "c-about-section__tertiary-heading u-bottom-margin-small",
                },
                "ref": null,
                "rendered": "I'm Chidi Njoku and I'm a full stack web developer",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "I am a passionate and talented developer. I posess strong coding, technical, problem solving abilities and a knack for writting clean, readable code. I am the kind of guy who goes beyond the routine and likes to do a little bit of everything. I am committed to collaborative problem solving, building quality products and sophisticated design.",
                  "className": "u-bottom-margin-medium",
                },
                "ref": null,
                "rendered": "I am a passionate and talented developer. I posess strong coding, technical, problem solving abilities and a knack for writting clean, readable code. I am the kind of guy who goes beyond the routine and likes to do a little bit of everything. I am committed to collaborative problem solving, building quality products and sophisticated design.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Summary of Skills",
                  "className": "c-about-section__tertiary-heading",
                },
                "ref": null,
                "rendered": "Summary of Skills",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <li
                      className="c-about-section__item"
                    >
                      Experience with Ruby on Rails
                    </li>,
                    <li
                      className="c-about-section__item"
                    >
                      Solid understanding of SQL
                    </li>,
                    <li
                      className="c-about-section__item"
                    >
                      Experince with AWS and Docker containers
                    </li>,
                    <li
                      className="c-about-section__item"
                    >
                      Experince with Test-Driven Development (TDD), specifically Rspec/Capybara and Jest/Enzyme
                    </li>,
                    <li
                      className="c-about-section__item"
                    >
                      Experience with conitnuous integration
                    </li>,
                    <li
                      className="c-about-section__item"
                    >
                      Experience with GraphQl, React.js
                    </li>,
                  ],
                  "className": "c-about-section__list",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Experience with Ruby on Rails",
                      "className": "c-about-section__item",
                    },
                    "ref": null,
                    "rendered": "Experience with Ruby on Rails",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Solid understanding of SQL",
                      "className": "c-about-section__item",
                    },
                    "ref": null,
                    "rendered": "Solid understanding of SQL",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Experince with AWS and Docker containers",
                      "className": "c-about-section__item",
                    },
                    "ref": null,
                    "rendered": "Experince with AWS and Docker containers",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Experince with Test-Driven Development (TDD), specifically Rspec/Capybara and Jest/Enzyme",
                      "className": "c-about-section__item",
                    },
                    "ref": null,
                    "rendered": "Experince with Test-Driven Development (TDD), specifically Rspec/Capybara and Jest/Enzyme",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Experience with conitnuous integration",
                      "className": "c-about-section__item",
                    },
                    "ref": null,
                    "rendered": "Experience with conitnuous integration",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Experience with GraphQl, React.js",
                      "className": "c-about-section__item",
                    },
                    "ref": null,
                    "rendered": "Experience with GraphQl, React.js",
                    "type": "li",
                  },
                ],
                "type": "ul",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="u-center-text u-bottom-margin-small"
          >
            <h2
              className="c-about-section__primary-heading"
            >
              About Me
            </h2>
          </div>,
          <div
            className="c-about-section__text-image-container"
          >
            <div
              className="c-image-composition"
            >
              <img
                alt="photo-1"
                className="c-image-composition__photo c-image-composition__photo--p1"
                src="/static/images/pic22.jpg"
              />
              <img
                alt="photo-2"
                className="c-image-composition__photo c-image-composition__photo--p2"
                src="/static/images/pic6.jpg"
              />
              <img
                alt="photo-3"
                className="c-image-composition__photo c-image-composition__photo--p3"
                src="/static/images/pic20.jpg"
              />
            </div>
            <div
              className="c-about-section__about-me"
            >
              <h3
                className="c-about-section__tertiary-heading u-bottom-margin-small"
              >
                I'm Chidi Njoku and I'm a full stack web developer
              </h3>
              <p
                className="u-bottom-margin-medium"
              >
                I am a passionate and talented developer. I posess strong coding, technical, problem solving abilities and a knack for writting clean, readable code. I am the kind of guy who goes beyond the routine and likes to do a little bit of everything. I am committed to collaborative problem solving, building quality products and sophisticated design.
              </p>
              <h3
                className="c-about-section__tertiary-heading"
              >
                Summary of Skills
              </h3>
              <ul
                className="c-about-section__list"
              >
                <li
                  className="c-about-section__item"
                >
                  Experience with Ruby on Rails
                </li>
                <li
                  className="c-about-section__item"
                >
                  Solid understanding of SQL
                </li>
                <li
                  className="c-about-section__item"
                >
                  Experince with AWS and Docker containers
                </li>
                <li
                  className="c-about-section__item"
                >
                  Experince with Test-Driven Development (TDD), specifically Rspec/Capybara and Jest/Enzyme
                </li>
                <li
                  className="c-about-section__item"
                >
                  Experience with conitnuous integration
                </li>
                <li
                  className="c-about-section__item"
                >
                  Experience with GraphQl, React.js
                </li>
              </ul>
            </div>
          </div>,
        ],
        "className": "c-about-section",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h2
              className="c-about-section__primary-heading"
            >
              About Me
            </h2>,
            "className": "u-center-text u-bottom-margin-small",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "About Me",
              "className": "c-about-section__primary-heading",
            },
            "ref": null,
            "rendered": "About Me",
            "type": "h2",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="c-image-composition"
              >
                <img
                  alt="photo-1"
                  className="c-image-composition__photo c-image-composition__photo--p1"
                  src="/static/images/pic22.jpg"
                />
                <img
                  alt="photo-2"
                  className="c-image-composition__photo c-image-composition__photo--p2"
                  src="/static/images/pic6.jpg"
                />
                <img
                  alt="photo-3"
                  className="c-image-composition__photo c-image-composition__photo--p3"
                  src="/static/images/pic20.jpg"
                />
              </div>,
              <div
                className="c-about-section__about-me"
              >
                <h3
                  className="c-about-section__tertiary-heading u-bottom-margin-small"
                >
                  I'm Chidi Njoku and I'm a full stack web developer
                </h3>
                <p
                  className="u-bottom-margin-medium"
                >
                  I am a passionate and talented developer. I posess strong coding, technical, problem solving abilities and a knack for writting clean, readable code. I am the kind of guy who goes beyond the routine and likes to do a little bit of everything. I am committed to collaborative problem solving, building quality products and sophisticated design.
                </p>
                <h3
                  className="c-about-section__tertiary-heading"
                >
                  Summary of Skills
                </h3>
                <ul
                  className="c-about-section__list"
                >
                  <li
                    className="c-about-section__item"
                  >
                    Experience with Ruby on Rails
                  </li>
                  <li
                    className="c-about-section__item"
                  >
                    Solid understanding of SQL
                  </li>
                  <li
                    className="c-about-section__item"
                  >
                    Experince with AWS and Docker containers
                  </li>
                  <li
                    className="c-about-section__item"
                  >
                    Experince with Test-Driven Development (TDD), specifically Rspec/Capybara and Jest/Enzyme
                  </li>
                  <li
                    className="c-about-section__item"
                  >
                    Experience with conitnuous integration
                  </li>
                  <li
                    className="c-about-section__item"
                  >
                    Experience with GraphQl, React.js
                  </li>
                </ul>
              </div>,
            ],
            "className": "c-about-section__text-image-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt="photo-1"
                    className="c-image-composition__photo c-image-composition__photo--p1"
                    src="/static/images/pic22.jpg"
                  />,
                  <img
                    alt="photo-2"
                    className="c-image-composition__photo c-image-composition__photo--p2"
                    src="/static/images/pic6.jpg"
                  />,
                  <img
                    alt="photo-3"
                    className="c-image-composition__photo c-image-composition__photo--p3"
                    src="/static/images/pic20.jpg"
                  />,
                ],
                "className": "c-image-composition",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "photo-1",
                    "className": "c-image-composition__photo c-image-composition__photo--p1",
                    "src": "/static/images/pic22.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "photo-2",
                    "className": "c-image-composition__photo c-image-composition__photo--p2",
                    "src": "/static/images/pic6.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "photo-3",
                    "className": "c-image-composition__photo c-image-composition__photo--p3",
                    "src": "/static/images/pic20.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="c-about-section__tertiary-heading u-bottom-margin-small"
                  >
                    I'm Chidi Njoku and I'm a full stack web developer
                  </h3>,
                  <p
                    className="u-bottom-margin-medium"
                  >
                    I am a passionate and talented developer. I posess strong coding, technical, problem solving abilities and a knack for writting clean, readable code. I am the kind of guy who goes beyond the routine and likes to do a little bit of everything. I am committed to collaborative problem solving, building quality products and sophisticated design.
                  </p>,
                  <h3
                    className="c-about-section__tertiary-heading"
                  >
                    Summary of Skills
                  </h3>,
                  <ul
                    className="c-about-section__list"
                  >
                    <li
                      className="c-about-section__item"
                    >
                      Experience with Ruby on Rails
                    </li>
                    <li
                      className="c-about-section__item"
                    >
                      Solid understanding of SQL
                    </li>
                    <li
                      className="c-about-section__item"
                    >
                      Experince with AWS and Docker containers
                    </li>
                    <li
                      className="c-about-section__item"
                    >
                      Experince with Test-Driven Development (TDD), specifically Rspec/Capybara and Jest/Enzyme
                    </li>
                    <li
                      className="c-about-section__item"
                    >
                      Experience with conitnuous integration
                    </li>
                    <li
                      className="c-about-section__item"
                    >
                      Experience with GraphQl, React.js
                    </li>
                  </ul>,
                ],
                "className": "c-about-section__about-me",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "I'm Chidi Njoku and I'm a full stack web developer",
                    "className": "c-about-section__tertiary-heading u-bottom-margin-small",
                  },
                  "ref": null,
                  "rendered": "I'm Chidi Njoku and I'm a full stack web developer",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "I am a passionate and talented developer. I posess strong coding, technical, problem solving abilities and a knack for writting clean, readable code. I am the kind of guy who goes beyond the routine and likes to do a little bit of everything. I am committed to collaborative problem solving, building quality products and sophisticated design.",
                    "className": "u-bottom-margin-medium",
                  },
                  "ref": null,
                  "rendered": "I am a passionate and talented developer. I posess strong coding, technical, problem solving abilities and a knack for writting clean, readable code. I am the kind of guy who goes beyond the routine and likes to do a little bit of everything. I am committed to collaborative problem solving, building quality products and sophisticated design.",
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Summary of Skills",
                    "className": "c-about-section__tertiary-heading",
                  },
                  "ref": null,
                  "rendered": "Summary of Skills",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <li
                        className="c-about-section__item"
                      >
                        Experience with Ruby on Rails
                      </li>,
                      <li
                        className="c-about-section__item"
                      >
                        Solid understanding of SQL
                      </li>,
                      <li
                        className="c-about-section__item"
                      >
                        Experince with AWS and Docker containers
                      </li>,
                      <li
                        className="c-about-section__item"
                      >
                        Experince with Test-Driven Development (TDD), specifically Rspec/Capybara and Jest/Enzyme
                      </li>,
                      <li
                        className="c-about-section__item"
                      >
                        Experience with conitnuous integration
                      </li>,
                      <li
                        className="c-about-section__item"
                      >
                        Experience with GraphQl, React.js
                      </li>,
                    ],
                    "className": "c-about-section__list",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Experience with Ruby on Rails",
                        "className": "c-about-section__item",
                      },
                      "ref": null,
                      "rendered": "Experience with Ruby on Rails",
                      "type": "li",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Solid understanding of SQL",
                        "className": "c-about-section__item",
                      },
                      "ref": null,
                      "rendered": "Solid understanding of SQL",
                      "type": "li",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Experince with AWS and Docker containers",
                        "className": "c-about-section__item",
                      },
                      "ref": null,
                      "rendered": "Experince with AWS and Docker containers",
                      "type": "li",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Experince with Test-Driven Development (TDD), specifically Rspec/Capybara and Jest/Enzyme",
                        "className": "c-about-section__item",
                      },
                      "ref": null,
                      "rendered": "Experince with Test-Driven Development (TDD), specifically Rspec/Capybara and Jest/Enzyme",
                      "type": "li",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Experience with conitnuous integration",
                        "className": "c-about-section__item",
                      },
                      "ref": null,
                      "rendered": "Experience with conitnuous integration",
                      "type": "li",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Experience with GraphQl, React.js",
                        "className": "c-about-section__item",
                      },
                      "ref": null,
                      "rendered": "Experience with GraphQl, React.js",
                      "type": "li",
                    },
                  ],
                  "type": "ul",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
